src/panels/lovelace/hui-unused-entities.ts

```diff
@@ -3,26 +3,39 @@ import {
   LitElement,
   PropertyDeclarations,
   TemplateResult,
+  PropertyValues,
 } from "lit-element";
 
 import "./cards/hui-entities-card";
 
+import "@polymer/paper-fab/paper-fab";
```

Comment:
Let's use https://material-components.github.io/material-components-web-components/demos/fab.html

Reply:
`mwc` uses a font for the icons, I couldn't think of an easy way to fix that besides including their (or mdi) font...